nav2_bringup/launch/slam_launch.py

```diff
@@ -77,22 +74,22 @@ def generate_launch_description():
 
     # Nodes launching commands
 
-    start_map_saver_server_cmd = Node(
+    start_map_server = GroupAction(
+        actions=[SetParameter("use_sim_time", use_sim_time), Node(
```

Comment:
Set new line for the `Node` start

Reply:
The formatting here is a little funky for the `start_map_server` action

Reply:
Yeah I know it's funky but it passes the tests, I still don't understand why we don't have an automatic python formatter in ament_lint, like we have ament_uncrustify. I found this [discussion](https://discourse.ros.org/t/python-black-formatting/13277/27?page=2) about it.

Do you use any automatic python formatter?

Reply:
Do you usually fix those manually? Does ROS not have an auto-formatter in ament_lint?